### PR TITLE
chore: use informalsystems fork of `tendermint`

### PR DESCRIPTION
**Pull request by aljo242**
Use Informal Systems [fork](https://github.com/informalsystems/tendermint) of Tendermint.

[Context](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.8)

### CODE REVIEW

File: chain/go.mod [hunk 1]
```diff
@@ -292,3 +292,8 @@ require (
 	pgregory.net/rapid v0.5.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace (
+	// use informal system fork of tendermint
+	github.com/tendermint/tendermint => github.com/informalsystems/tendermint v0.34.24
```
Comment (gjermundgaraba) [inline chain/go.mod:298]: I think this needs to be [v0.37.0-rc2](https://github.com/informalsystems/tendermint/releases/tag/v0.37.0-rc2) for this to work

**Review by gjermundgaraba — CHANGES_REQUESTED** [review]

**Comment by gjermundgaraba** [comment]
This is awesome! I've thought about this and assumed this had to be done pretty soon!

File: chain/go.mod (hunk 1 above)
Comment (aljo242) [reply chain/go.mod]: https://github.com/EmpowerPlastic/empowerchain/pull/460/commits/5c17c477ab48a50d502e42a428a2eb21f9e822db

**Review by gjermundgaraba — APPROVED** [review]

**Comment by gjermundgaraba** [comment]
Sweet! Thanks @aljo242 :rocket:

**Pull request merged**